sklearn/linear_model/base.py

```diff
@@ -112,6 +112,19 @@ def center_data(X, y, fit_intercept, normalize=False, copy=True,
     return X, y, X_mean, y_mean, X_std
 
 
+def rescale_data(X, y, sample_weight):
```

Comment:
`rescale_data` is used in ridge.py as well. Though in some sense, it is private to linear_model.